thrusters_blue_robotics_t500Types.hpp

```diff
@@ -1,8 +1,10 @@
 #ifndef thrusters_blue_robotics_t500_TYPES_HPP
 #define thrusters_blue_robotics_t500_TYPES_HPP
 
+#include <base/JointState.hpp>
```

Comment:
unecessary include(maybe itll be used in the future, not sure)

Reply:
just a left over, tks